src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -115,7 +115,10 @@ export const BlockPreviewBase = ({
         blockId={block.id}
         studioEndpointUrl={getConfig().STUDIO_BASE_URL}
         lmsEndpointUrl={getConfig().LMS_BASE_URL}
-        returnFunction={() => () => setShowEditorModal(false)}
+        returnFunction={() => () => {
+          setShowEditorModal(false);
+          setUpdatedBlock(true);
+        }}
       />
```

Comment:
returnFunction does get passed something (to show if it was a save or a cancel), and we could use that to choose to update the block or not.

Reply:
in fact, the parameter to returnFunction is the response from the save call. (in a different world, we could just use that data to reload the block, but it's not the student_view)